website/docs/r/dashboard.html.markdown

```diff
@@ -0,0 +1,824 @@
+- `y` - (Required) The position of the widget on the y (horizontal) axis. Should be greater or equal to 0.
+- `width` - (Required) The width of the widget.
+- `height` - (Required) The height of the widget.
+
+### Nested `widget` `axis` blocks
+Nested `axis` blocks have the following structure:
+
+- `label` - (Optional) The label of the axis to display on the graph.
+- `scale` - (Optional) Specifies the scale type. One of "linear", "log", "pow", "sqrt".
+- `min` - (Optional) Specify the minimum value to show on y-axis.
+- `max` - (Optional) Specify the minimum value to show on y-axis.
+- `include_zero` - (Optional) Always include zero or fit the axis to the data range.
+
+### Nested `widget` `conditional_formats` blocks
+Nested `conditional_formats` blocks have the following structure:
+
+- `comparator` - (Required) Comparator to apply from: One of `>`, `>=`, `<`, or `<=`.
+- `value` - (Required) Value for the comparator.
+- `palette` - (Required) Color palette to apply; One of `blue`, `custom_bg`, `custom_image`, `custom_text`, `gray_on_white`, `green`, `green_on_white`, `grey`, `orange`, `red`, `red_on_white`, `white_on_gray`, `white_on_green`, `white_on_red`, `white_on_yellow`, or `yellow_on_white`.
+- `custom_bg_color` - (Optional) Color palette to apply to the background, same values available as palette.
+- `custom_fg_color` - (Optional) Color palette to apply to the foreground, same values available as palette.
+- `image_url` - (Optional) Displays an image as the background.
+.
+
+### Nested `widget` `time` blocks
+Nested `widget` `time` blocks have the following structure:
+
+- `live_span` - (Required) The timeframe to use when displaying the widget. One of `10m`, `30m`, `1h`, `4h`, `1d`, `2d`, `1w`.
+
+### Nested `apm_query` and `log_query` blocks
+Nested `apm_query` and `log_query` blocks have the following structure (Visit the [ Graph Primer](https://docs.datadoghq.com/graphing/) for more information about these values):
+
+  - `index` - (Required)
+  - `compute` - (Required). Exactly one nested block is required with the following structure:
+    - `aggregation` - (Required)
+    - `facet` - (Optional)
+    - `interval` - (Optional)
+  - `search` - (Optional). One nested block is allowed with the following structure:
+    - `query` - (Optional)
+  - `group_by` - (Optional). Multiple nested blocks are allowed with the following structure:
+    - `facet` - (Optional)
+    - `limit` - (Optional)
+    - `sort` - (Optional). One nested block is allowed with the following structure:
+      - `aggregation` - (Optional)
+      - `order` - (Optional)
+      - `facet` - (Optional)
+
+
+### Nested  `process_query` blocks
+Nested `process_query` blocks have the following structure (Visit the [ Graph Primer](https://docs.datadoghq.com/graphing/) for more information about these values):
+
+  - `metric` - (Required)
+  - `search_by` - (Required)
+  - `filter_by` - (Required)
+  - `limit` - (Required)
+
+
+### Nested `widget``marker` blocks
```

Comment:
Missing a space here between `widget` and `marker`, would result in bad formatting.